timelapse/recorder.py

```diff
@@ -2,7 +2,16 @@
 import subprocess  # Taking screenshot
 import datetime  # Filename
 from multiprocessing import Process, Event
+from PyObjCTools import AppHelper
+from AppKit import *
 
+def getScreenIndexWithMouse():
```

Comment:
I'm not sure about this project's naming conventions, but I believe PEP8 requires function names to lower case and `_` separated.